ocs_ci/framework/tests/test_main.py

```diff
@@ -52,26 +52,25 @@ def test_config_passing(self, config_update, pytest_main, testdir):
                 os.path.join(tempdir, "ocsci_conf.yaml"),
             ]
         )
-        assert config_update.call_args_list == [
-            mock.call(dict(RUN=None)),
-        ]
+        assert config_update.call_args_list[0] == mock.call(dict(RUN=None))
 
     @mock.patch("ocs_ci.framework.main.pytest.main")
-    @mock.patch.object(config, "update")
+    @mock.patch.object(config, "update", wraps=config.update)
     def test_multi_config_passing(self, config_update, pytest_main, testdir):
         tempdir = testdir.makefile(
             ".yaml",
             ocsci_conf1="RUN: null",
             ocsci_conf2="TEST_SECTION: null",
         ).dirname
-        main.main(
-            [
-                "--ocsci-conf",
-                f"{os.path.join(tempdir, 'ocsci_conf1.yaml')}",
-                "--ocsci-conf",
-                f"{os.path.join(tempdir, 'ocsci_conf2.yaml')}",
-            ]
-        )
+        with raises(ValueError):
```

Comment:
Why does the main function return a ValueError with these args now? Shouldn't this be a legitimate use case?